community/libgit2/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+(
+    mkdir -p build
+
+    cd build
+
+    cmake .. \
+        -DCMAKE_BUILD_TYPE=None \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_INSTALL_LIBDIR=/usr/lib \
+        -DBUILD_SHARED_LIBS=OFF
+
+    make
+)
+
+make -C build DESTDIR="$1" install
```

Comment:
cmake builds should follow: https://k1ss.org/wiki/kiss/style-guide#0800

Also see: #1351